bootstrap-app/README.md

```diff
@@ -87,6 +87,22 @@ HTTP/1.1 200 OK
 {"message":"Rebalancing completed for collection C5-PROV1"}
 ```
 
+## Resharding Indexes
+
+### Start Resharding
+Starts the resharding process to create a new index with the given number of shards and copies
+the data from the old index to the new index. Both indexes will be used for ingest/search until
```

Comment:
Both indexes will be used for ingest, but the new index shouldn't be used for search.